delegates.rb

```diff
@@ -12,7 +12,13 @@
   $semaphore.synchronize {
     # Might have been done in another thread.
     if $logger.nil?
-      $logger = Java::edu.illinois.library.cantaloupe.script.Logger
+      # XXX: Cantaloupe 5 renamespace'd the logger, so try to load it from
+      # the new namespace, first.
+      begin
+        $logger = Java::edu.illinois.library.cantaloupe.delegate.Logger
+      rescue
+        $logger = Java::edu.illinois.library.cantaloupe.script.Logger
+      end
```

Comment:
A note to myself: We're making the break with the `default_i8_delegates.rb` stuff, might as well just move over to the new namespace.